style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3e3c3c;
+  padding: 0.7rem;
+  width: 100%;
+}
+
+body {
+  background-color: #1c1a19;
+}
+
+h1 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 45px;
+  color: #f55800;
+  width: 300px;
+  line-height: 1em;
+}
+
+h4 {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 19px;
+  color: #fff;
+  line-height: 3em;
+}
+
+p {
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: #fff;
+  width: auto;
+  letter-spacing: 0.01rem;
+}
+
+.name {
+  padding-right: 1rem;
+}
+
+.na {
+  color: #fff;
+  font-weight: 600;
+  font-size: 1rem;
+  line-height: 20px;
+  font-family: 'poppins', sans-serif;
+}
+
+.na a {
+  color: #fff;
+  font-family: 'poppins', sans-serif;
+  font-size: 1rem;
+  line-height: 5px;
+  text-decoration: none;
+  font-weight: 500;
+}
+
+.na li {
+  margin-left: 1rem;
+  list-style-type: none;
+}
+
+.icons {
+  padding-left: 0.5rem;
+  list-style-type: none;
+  display: flex;
+  margin-top: 1rem;
+}
+
+.icons li {
+  padding-right: 1.2rem;
+}
+
+.ham img {
+  margin-right: 1rem;
+}
+
+.image {
+  position: absolute;
+  bottom: 3.5rem;
+  right: 0;
+}
+
+#headlines {
+  background-image: url(image/illustration-header-mobile.png);
+  background-position: right 10.5em;
+  background-repeat: no-repeat;
+  background-size: 8em;
+  height: 37em;
+  padding: 5rem 2.5rem 0.5rem 1.5rem;
+}
```

Comment:
- [ ] As requested by the other reviewers, please use flexbox to layout the contents of the headline section. This is best for responsiveness. You can use `display: flex; flex-direction:column;`.

Reply:
@Kamba56 thanks for the corrections. I added a flex property to my work as requested